normalizer/tests/test_https_endpoint.py

```
@@ -0,0 +1,173 @@
+        test_https_availability()
+        
+        # 具体的な成功条件をチェック
+        # 少なくとも1つのエンドポイントが応答する必要がある
+        endpoints_tested = 0
+        successful_responses = 0
+        
+        test_address = "東京都中央区京橋1丁目19番13号"
+        http_base = base_url.replace("https://", "http://")
+        https_base = base_url.replace("http://", "https://")
+        
+        for endpoint_name, endpoint_data in [
```

Ruff (0.12.2)

Loop control variable `endpoint_name` not used within loop body

Rename unused `endpoint_name` to `_endpoint_name`

(B007)